### DISCUSSION TITLE
CPU Power Slider not showing

### DISCUSSION THREAD

**Discussion by phillip3p** [seerge/g-helper#966] [Q&A] [answered]
Hello,

First off, thank you for making this app. It allowed me to finally be able to charge my computer without issues. Second, I was trying to look into the CPU power sliders, but they are not showing on my computer. The only option I am getting under the CPU tab is for the boost. I have a Zephyrus M15 with an Intel i7-10750H and an NVIDIA GeForce RTX 2060. I'm also not getting the different options for the GPU, but based on the previous discussions I believe that is working fine.

**Comment by phillip3p** [comment]
Sorry, also forgot to mention temp limits. Those are not showing for either chip.

**Comment by seerge** [comment, marked as answer]
@phillip3p hello, if you don't see a slider under CPU section - it means it's not exposed by your device hardware / doesn't exist. So app just hides unsupported features. I assume you also don't have power limits in Armoury Crate?

**Reply by phillip3p** [reply]
I don't believe I did. So I'm guessing that applies to the temperature control as well?

**Reply by seerge** [reply]
Temp control / undervolting is for AMD only.

But you can try ThrottleStop utility https://www.techpowerup.com/download/techpowerup-throttlestop/ it works for intels :)

**Reply by phillip3p** [reply]
Awesome! Thank you so much. It really is a great software. Couldn't charge my laptop without it. Armory Crate was doing all sorts of weird stuff to my charger control and the computer kept crashing. Ever since I installed it I've had no issues
